From: Hrvoje Senjan <hrvoje.senjan@gmail.com>
Date: Thu, 28 May 2015 15:56:47 +0000
Subject: Yet another _include_dirs fix
X-Git-Url: http://quickgit.kde.org/?p=phonon.git&a=commitdiff&h=635b65fa417f49ac4ae189e926bf138efc6544d6
---
Yet another _include_dirs fix

The variable is set as a definition, so mark it as such
---


--- a/cmake/FindPhononInternal.cmake
+++ b/cmake/FindPhononInternal.cmake
@@ -409,7 +409,7 @@
       file(WRITE "${_source_file}" "${_source}")
       set(_include_dirs "-DINCLUDE_DIRECTORIES:STRING=${QT_INCLUDES}")
 
-      try_compile(_compile_result ${CMAKE_BINARY_DIR} ${_source_file} CMAKE_FLAGS "${_include_dirs}" OUTPUT_VARIABLE _compile_output_var)
+      try_compile(_compile_result ${CMAKE_BINARY_DIR} ${_source_file} CMAKE_FLAGS "${CMAKE_CXX_FLAGS}" COMPILE_DEFINITIONS "${_include_dirs}" OUTPUT_VARIABLE _compile_output_var)
 
       if(NOT _compile_result)
          message("${_compile_output_var}")

